lib/ret_web/plugs/admin_only.ex

```diff
@@ -12,7 +12,8 @@ defmodule RetWeb.Plugs.AdminOnly do
       conn
     else
       conn
-      |> put_status(:unauthorized)
+      |> resp(401, "Not authorized")
+      |> send_resp()
```

Comment:
Not sure what the difference between them is, but I think you can do this in one line with 
`|> send_resp(401, "Not authorized")`